code/modules/exploration_crew/exploration_explosives.dm

```diff
@@ -0,0 +1,126 @@
+/obj/item/grenade/exploration
+	name = "breaching charges"
+	desc = "Used by exploration crews to breach and demolish unknown places. Requires a nearby detonator to be triggered."
+	icon_state = "plastic-explosive0"
+	item_state = "plastic-explosive"
+	lefthand_file = 'icons/mob/inhands/weapons/bombs_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/bombs_righthand.dmi'
+	item_flags = NOBLUDGEON
+	flags_1 = NONE
+	w_class = WEIGHT_CLASS_SMALL
+	var/atom/target = null
+	var/mutable_appearance/plastic_overlay
+	var/boom_sizes = list(0, 0, 3)
+	var/list/attached_detonators = list()
+
+/obj/item/grenade/exploration/Initialize()
+	. = ..()
+	plastic_overlay = mutable_appearance(icon, "[item_state]2", HIGH_OBJ_LAYER)
+
+/obj/item/grenade/exploration/Destroy()
+	for(var/obj/item/exploration_detonator/detonator in attached_detonators)
+		detonator.linked_explosives -= src
+	attached_detonators = null
+	. = ..()
+
+/obj/item/grenade/exploration/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/exploration_detonator))
+		var/obj/item/exploration_detonator/detonator = W
+		detonator.linked_explosives |= src
+		attached_detonators |= detonator
+		to_chat(user, "<span class='notice'>You link [src] to [W].</span>")
```

Comment:
all those would work better as balloon alerts but whatever

Reply:
Text that is long looks awful in balloon alerts. I guess having it as (device linked!) would work.